sphinx/source/docs/user_guide/data.rst

```diff
@@ -241,27 +338,35 @@ return a JavaScript array with the transformed values:
     plot.line(x='aapl_date', y=transform('aapl_close', normalize), line_width=2,
               color='#cf3c4d', alpha=0.6,legend="Apple", source=aapl_source)
 
-The above code converts raw price data into a sequence of normalized returns
-relative to the first data point. The full result is shown below:
+The code in this example converts raw price data into a sequence of normalized
+returns that are relative to the first data point:
 
 .. bokeh-plot:: docs/user_guide/examples/data_transforming_customjs_transform.py
     :source-position: none
 
+.. _userguide_data_filtering:
 
-Filtering Data
+Filtering data
 --------------
 
-It's often desirable to focus in on a portion of data that has been subsampled or filtered
-from a larger dataset. Bokeh allows you to specify a view of a data source that represents
-a subset of data. By having a view of the data source, the underlying data doesn't need to
-be changed and can be shared across plots. The view consists of one or more filters that
-select the rows of the data source that should be bound to a specific glyph.
+Bokeh uses a concept called "view" to select subsets of data. Views are
+represented by Bokeh's |CDSView| class. When you use a view, you can use one or
+more filters to select specific data points without changing the underlying
+data. You can also share those views between different plots.
+
+To plot with a filtered subset of data, pass a |CDSView| to the ``view``
+argument of any renderer methods that are part of Bokeh's |Figure| class.
```

Comment:
I would suggest "renderer methods on a Bokeh plot". Generally I try to avoid linking to capital-F figure since it's not typically what users use (they use `figure`, or `Plot` if they are working with low level models)